src/main/resources/db/migration/V1_3__create_media_table.sql

```diff
@@ -1,18 +1,20 @@
 CREATE TABLE IF NOT EXISTS moj_media (
-  moj_med_id               INTEGER                     NOT NULL
-, moj_crt_id               INTEGER
-, r_media_object_id        text
-, c_channel                INTEGER
-, c_total_channels         INTEGER
-, c_reference_id           text
-, c_start                  TIMESTAMP without TIME ZONE
-, c_end                    TIMESTAMP without TIME ZONE
-, c_courtroom              text
-, c_reporting_restrictions INTEGER
-, c_case_id                text
-, r_case_object_id         text
-, r_version_label          text
-, i_superseded             boolean
-, i_version                INTEGER
-, CONSTRAINT moj_media_pkey PRIMARY KEY (moj_med_id)
-)
+  moj_med_id              INTEGER                     NOT NULL
+, moj_ctr_id              INTEGER
+, r_media_object_id       CHARACTER VARYING(16)
+, c_channel               INTEGER
+, c_total_channels        INTEGER
+, c_reference_id          CHARACTER VARYING
+, c_start                 TIMESTAMP WITH TIME ZONE
+, c_end                   TIMESTAMP WITH TIME ZONE
+, c_case_id               CHARACTER VARYING(32) ARRAY
+, r_case_object_id        CHARACTER VARYING(16) ARRAY
+, r_version_label         CHARACTER VARYING(32)
+, i_superseded            BOOLEAN
+, i_version               INTEGER
+, CONSTRAINT moj_media_pk PRIMARY KEY (moj_med_id)
+--, CONSTRAINT moj_media_courtroom_fk FOREIGN KEY (moj_ctr_id) REFERENCES moj_courtroom (moj_ctr_id)
```

Comment:
Intentional to leave this commented?

Reply:
Yes, I'm afraid it has to be commented out at this position in the flyway scripts because we don't have `moj_courtroom ` until later on in `V1_13__Create_case_data_tables.sql` so I've added the FK constraint there instead
`ALTER TABLE moj_media ADD CONSTRAINT moj_media_courtroom_fk FOREIGN KEY (moj_ctr_id) REFERENCES moj_courtroom(moj_ctr_id);`

However, these dev flyway scripts may be short lived if they are to be replaced with Barry's v29+ modernised schema design instead